chapters/en/chapter2.md

```diff
@@ -11,7 +11,7 @@ type: chapter
 id: 2
```

Comment:
Sorry to make this comment here but I noticed in the chapter's description that there is an extra apostrophe in `You''ll learn how to make the most...`

```suggestion
  "In this chapter, you'll use your new skills to extract specific information
  from large volumes of text. You'll learn how to make the most of spaCy's data
  structures, and how to effectively combine statistical and rule-based
  approaches for text analysis."
```